app/views/events/_event.html.erb

```diff
@@ -0,0 +1,11 @@
+<a href="/event">
+    <%= render "components/eventbox",
+        :title => event.eventName,
+        :datetime => "#{event.startDate} at 17:00 - 18:30",
+        :description => event.description,
+        :schooloruni => false,
```

Comment:
I'm leaving these as is since its Craigs partial (hence the hash syntax) - there's a bunch of these needing the attributes to be updated but I was going to worry about that after the designs have settled down.